File: Fase1/2014/nivel1/triangulo.py
def is_triangulo(a, b ,c):
    if a < (b+c) and b < (a+c) and c < (b+a):
        return True
    return False


#pego os 4 comprimentos que são solicitados na questão
n1, n2, n3, n4 = [int(n) for n in input().split()]

if (is_triangulo(n1, n2, n3) or 
is_triangulo(n1, n2, n4) or 
is_triangulo(n1, n3, n4) or 
is_triangulo(n2, n3, n4)):
    print('S')
else:
    print('N')

File: Fase1/2014/nivel1/triangulo.js
// Função para verificar se é triângulo
function isTriangulo(a, b, c) {
    if (a < (b + c) && b < (a + c) && c < (b + a)) {
        return true;
    }
    return false;
}

// Obtendo os 4 comprimentos que são solicitados na questão
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Digite os 4 comprimentos separados por espaço: ", function (input) {
    const [n1, n2, n3, n4] = input.split(' ').map(Number);

    // Testando para todos os comprimentos
    if (isTriangulo(n1, n2, n3) || isTriangulo(n1, n2, n4) || isTriangulo(n1, n3, n4) || isTriangulo(n2, n3, n4)) {
        console.log('S');
    } else {
        console.log('N');
    }

    rl.close();
});
